db/migrate/20221202115620_add_user_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Again, considering the given ERD, the reference name should be `author` instead of `user`, hence creating a migration query should be similar to:

```ruby
...
  def change
    add_reference :comments, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

Reply:
@rloterh Thank you very much for your details Review Feedback. Much appreciated.